samcli/hook_packages/terraform/hooks/prepare/resource_linking.py

```diff
@@ -109,3 +108,43 @@ def _get_configuration_address(address: str) -> str:
         The address clean of indices
     """
     return re.sub(r"\[[^\[\]]*\]", "", address)
+
+
+def _resolve_module_output(module, output_name):
+    pass
+
+
+def _resolve_module_variable(module: TFModule, variable_name: str) -> List[Union[ConstantValue, ResolvedReference]]:
+    # return a list of the values that resolve the passed variable
+    # name in the input module.
+    results: List[Union[ConstantValue, ResolvedReference]] = []
+
+    var_value = module.variables[variable_name]
```

Comment:
can we used `get` instead of `[]`, and please raise an exception if it is not there